satis-config/govcms-develop.json

```diff
@@ -9,17 +9,11 @@
         { "type": "vcs", "url": "https://github.com/govcms/require-dev" }
     ],
     "require": {
-        "govcms/govcms": "1.x-dev",
+        "govcms/govcms": "dev-develop",
```

Comment:
this doesn't resolve - I preferred to use a branch reference here (e.g. dev-release/1.2.0) to reinforce that the develop branch can be pointed to a future state of master - given that it is the release strategy for GovCMS to merge to a branch for testing - so the next release/* branch becomes a de-facto "develop"